src/content/smart-bulletins/035.md

```diff
@@ -0,0 +1,68 @@
+
+### OMB Circular A-123, Appendix B Chapter 5.3.1 – Statistical Reporting –
+Requested information in the template attached below shall be submitted, at a minimum, on an annual fiscal year basis. Agency-provided information shall be furnished to CCCM no later than 90 calendar days after the end of the Fiscal Year (FY). The due date for agency/organization submission of FY2020 agency statistical reporting to CCCM is January 31, 2021. 
+
+The Statistical Reporting requirement applies to all business lines under the GSA SmartPay program however; the following guidance is for transit benefit issuing programs only. All other agencies should see Smart Bulletin No.030 for reporting guidance. Transit benefit programs must report the following items for each business line:
+
+- Count of accounts, as defined by GSA; 
+- Total number of accounts across the charge card program as compared to the prior reporting period;
+- Total charge card dollars spent; and 
+- Total refunds earned.
+
+Specific requirements for the purchase cards business line are as follows:
+
+Purchase cards used for transit benefits
+- Number of transit benefit accounts with limits over the statutory limit;
+- Ratio of the number of confirmed violations reported pursuant to P.L. 112-194 as compared to the number of valid transactions within the reporting period (GSA's Center for Charge Card Management will obtain confirmed violation information from OMB/OFFM).
+
+### OMB Circular A-123, Appendix B Chapter 5.3.2 – Narrative Reporting –
+For agencies/organizations subject to the CFO Act, the following information should be updated annually on a fiscal year basis and provided to CCCM no later than 90 calendar days after the end of each fiscal year: 
+- The date(s) of most recent and next scheduled independent review(s) (e.g., OIG audits and risk assessments) for all agency charge card programs;
+- A description of the current process for monitoring transit benefit use, including what reports the agency reviews and what actions are taken when a problem is discovered;
+- A description of any best practices the agency employs in transit cardmanagement; and
+- Any additional information regarding agency management and use of the purchase card for transit benefit programs, including innovative applications to streamline business processes, increased revenue generation from expanded use for payments, and creative internal controls which maintain program integrity, flexibility and ease of use.
+
+For FY2020 reports and beyond, agency narrative reports are due to CCCM NLT 90 calendar days after end of the fiscal year requirement as specified in the current version of OMB Circular A-123, Appendix B (January 31).
+
+## Action
+Transit benefit programs shall utilize the reporting templates attached below to submit the required agency/organization statistical and narrative reporting data. Program
```

Comment:
Transit benefit programs shall utilize the reporting templates attached below to submit the required agency/organization statistical and narrative reporting data. Program reports should be e-mailed to OMBA123AppBReporting@gsa.gov using the following file naming convention: AgencyName_OMB_A123_FiscalYear (e.g., GeneralServicesAdministration_OMB_A123_2019). Further instructions for report submission are provided in the templates. In the event any sensitive data is to be provided in these reports, agencies/organizations are reminded to encrypt the files and take any other security precautions as required by agency/organization policy.